src/05-derivatives/03-imaging.md

```diff
@@ -183,7 +183,7 @@ The following metadata fields apply to all segmentation files:
 
 | **Key name** | **Description**                                                                                                                        |
 | ------------ | -------------------------------------------------------------------------------------------------------------------------------------- |
-| Manual       | OPTIONAL. Boolean. Indicates if the segmenation was performed manually or via an automated process                                     |
+| Manual       | OPTIONAL. Boolean. Indicates if the segmentation was performed manually or via an automated process                                     |
```

Comment:
Guessing you didn't delete a space to compensate?

```suggestion
| Manual       | OPTIONAL. Boolean. Indicates if the segmentation was performed manually or via an automated process                                    |
```

Reply:
I did not, forgot it was necessary.